src/GettextTranslationSaver.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Armandsar\LaravelTranslationio;
+
+use Illuminate\Contracts\Foundation\Application;
+use Illuminate\Filesystem\Filesystem;
+use Gettext\Translations;
+
+class GettextTranslationSaver
+{
+    /**
+     * @var Application
+     */
+    private $application;
+    /**
+     * @var Filesystem
+     */
+    private $filesystem;
+
+    public function __construct(
+        Application $application,
+        FileSystem $fileSystem
+    )
+    {
+        $this->application = $application;
+        $this->filesystem = $fileSystem;
+        $this->config = $application['config']['translationio'];
+    }
+
+    public function call($locale, $poContent)
+    {
+        $localeDir = $this->localePath($locale);
+        $lcMessagesDir = $localeDir . DIRECTORY_SEPARATOR . 'LC_MESSAGES';
+        $poFile = $localeDir . DIRECTORY_SEPARATOR . 'app.po';
+        $moFile = $lcMessagesDir . DIRECTORY_SEPARATOR . 'app.mo';
+
+        if ( ! $this->filesystem->exists($localeDir)) {
+            $this->filesystem->makeDirectory($localeDir, 0777, true);
```

Comment:
Could pass 4th argument as true, to force creating of directory, wont need to check if it exists